packages/core/src/components/Header/Navigation/MenuItem/MenuItem.styles.tsx

```diff
@@ -42,16 +42,16 @@ export const MenuItemLi = styled(
   ...(!$selected &&
     !$isMenu && {
```

Comment:
Since the height is now `100%`, I think we should add something line `borderBottom: ${theme.header.selectedItemBorderBottomThickness} solid ${theme.header.backgroundColor}` here to have this white bottom border in DS5 when hovering an item. 

![Screenshot 2023-06-13 at 09 28 07](https://github.com/lumada-design/hv-uikit-react/assets/43220251/b3636cbc-104d-4a9b-9eaf-e32938e0f4a3)

Reply:
Yeah, I think that's a good idea. I'll update the PR.